config-templates/aws/aws-efs-csi-driver/1.2.2/metadata.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "1.2.2",
+  "enabled": "true",
+  "description": "AWS EFS CSI Driver version 1.2.2",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "aws"
+    },
+    {
+      "name": "offering-name",
+      "value": "aws-efs-csi-driver"
+    },
+    {
+      "name": "provider-url",
+      "value": "https://console.aws.amazon.com/support/home#/"
+    },
+    {
+      "name": "support-msg",
+      "value": "For support contact AWS"
+    },
+    {
+      "name": "provider-logo",
+      "value": "/aws-efs-logo"
```

Comment:
Create a `logo` dir and copy `aws-efs-csi-driver.png` under `logo`  refer to https://github.com/IBM/ibm-satellite-storage/tree/develop/config-templates/aws/aws-efs-csi-driver/1.0.0/logo
and update to "value": "logo/aws-efs-csi-driver.png"

Reply:
Addressed